highfidelity/hifi
# Zip file structures replicated in asset server

Other minor changes: URL in Blocks app updated to Poly, updated an unzip debug print

Test plan:
1. Download https://hifi-content.s3.amazonaws.com/elisalj/blocks/lego_obj.zip
2. Open Interface, then open the asset browser (Ctrl + Shift + A)
3. Drag "lego_obj.zip" into Interface
4. The model should load in front of you (ignore the lack of textures)
5. Check that in the asset browser there is a "lego_obj" folder; expanding it should reveal a hierarchical file structure that matches what's inside lego_obj.zip
6. Delete "lego_obj" from the asset server
7. Load "Blocks" from the marketplace and open
8. Search for "Lego Moon Buggy" and select this: https://poly.google.com/view/8kSHCCK66fa
9. Download the obj for that; the model should still appear in front of you with no textures
10. Check in the asset browser for a file "8kSHC...", it should have the same file structure as "lego_obj"
